pkg/apis/kubeflow.org/v1/pytorch_types.go

```diff
@@ -106,8 +108,6 @@ type ElasticPolicy struct {
 	// --rdzv_backend, --rdzv_endpoint, --rdzv_id are auto-assigned; any explicitly set values
 	// are ignored.
 	Standalone *bool `json:"standalone,omitempty"`
-	// Number of workers per node; supported values: [auto, cpu, gpu, int].
-	NProcPerNode *int32 `json:"nProcPerNode,omitempty"`
```

Comment:
Quickly feedback. We shouldn't remove the existing API field.
If we remove the existing API fields, CRD is broken, and then the training operator can not reconcile against the already running PyTorchJob.

cc: @johnugeorge

Reply:
Also, we shouldn't break API since our API version is v1. We can leave those improvements on the v2 API.

Reply:
Done.
That's OK, I was trying to remove it because it was a misspelling version.